client/src/main/java/io/lionweb/client/delta/messages/DeltaQueryResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.lionweb.client.delta.messages;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+
+public class DeltaQueryResponse {
```

Comment:
I think we need to specialize this for every query, as they return different information.

Example: `SignOn` returns participationId, but `ListPartitions` returns a list of partitions.

Reply:
Making this class abstract

Reply:
I made this class abstract and created concrete subclasses